.github/workflows/platform.updateReadMe.yml

```
@@ -19,11 +19,12 @@ env:
   pipelinePrincipalGitUserEmail: 'CARML@noreply.github.com'
   relativeScriptPath_SetReadMeModuleTable: 'utilities/tools/platform/Set-ReadMeModuleTable.ps1'
   relativeScriptPath_SetModuleOverviewTable: 'utilities/tools/platform/Set-ModuleOverviewTable.ps1'
+  relativeScriptPath_SetPlatformPipelinesTable: 'utilities/tools/platform/Set-PlatformPipelinesTable.ps1'
 
 jobs:
   job_update_readme:
     runs-on: ubuntu-20.04
-    name: 'Update module tables'
+    name: 'Update status tables'
```

Isn't this task specific to Module pipelines anymore? Shouldn't it be as below consistently with the new `'Update Platform Pipelines table'` task?
```suggestion
    name: 'Update Module Pipelines table'
```

---

Jap, I just renamed it as I figured the main purpose is the status display, much less a simple list. I renamed it now to your suggestion `Update Module Pipelines table`. It's missing the status again, but at least it shows 'pipelines' :D

---

We can keep Status as well, I'd just align the name between platform and module pipeline steps. Your call

---

it's good as is. No need to overcomplicate. Just wanted to explain myself